content/blog/hcl-vs-pulumi/index.md

```diff
@@ -0,0 +1,1551 @@
+---
+title: "Pulumi vs HCL: Understanding the Language Differences in Infrastructure as Code"
+allow_long_title: true
+date: 2024-07-09
+meta_desc: "Compare Pulumi and HCL for infrastructure as code. Discover how general-purpose languages boost productivity and collaboration cloud management."
+meta_image: meta.png
+authors:
```

Comment:
the new template has a section for adding in the linkedin and twitter copy in here. can you please add your suggested copy for the post either in this pull request or directly into the markdown file?

Reply:
done